lib/mergeBuffers.js

```diff
@@ -41,8 +41,19 @@ function mergeBuffers(gltf, bufferId) {
                         bufferView.buffer = bufferId;
                     }
                 }
-                buffersToMerge.push(buffer.extras._pipeline.source);
-                lengthSoFar += buffer.extras._pipeline.source.length;
+
+                var sourceBuffer = buffer.extras._pipeline.source;
+                var currentBufferLength = sourceBuffer.length;
+                if (currentBufferLength % 4)

```

Comment:
Is this 4 because of `Float32` based on the implementation note here? https://github.com/KhronosGroup/glTF/tree/master/specification/1.0#bufferview-and-accessor-byte-alignment

 It might be worthwhile to add a blurb summarizing why the 4-byte alignment is needed.